kumascript/src/live-sample.js

```diff
@@ -118,30 +118,20 @@ class LiveSampleID {
 
 function getLiveSampleIDs(slug, source) {
   // Given a slug and its raw source HTML, parses the source and returns
```

Comment:
The changes in this file aren't necessary for this fix. I just wanted to simplify this function knowing that we don't care about sample ID's to be extracted from other documents.